spec-main/api-web-contents-spec.ts

```diff
@@ -1534,7 +1534,7 @@ describe('webContents module', () => {
         }
         response
           .writeHead(401, { 'WWW-Authenticate': 'Basic realm="Foo"' })
-          .end()
+          .end('401')
```

Comment:
confused about why this was needed, until i saw the below test where you check the page content :)